src/html/components/wordpress-images.html

```diff
@@ -1,9 +1,9 @@
 <h3 id="style-guide-wp-images" class="style-guide-title">WordPress Images &amp;&nbsp;Captions</h3>
 
-<p><figure id="attachment_2" class="wp-caption alignleft" style="width:150px"><img class="wp-image-1" src="./../assets/images/150x150.png" /><figcaption class="wp-caption-text">This is a left aligned caption</figcaption></figure> The rest of this paragraph is filler for the sake of seeing the text wrap around the 150×150 image, which is left aligned.</p>
+<p><figure id="attachment_2" class="wp-caption alignleft" style="width:150px"><img alt="" class="wp-image-1" src="./../assets/images/150x150.png" /><figcaption class="wp-caption-text">This is a left aligned caption</figcaption></figure> The rest of this paragraph is filler for the sake of seeing the text wrap around the 150×150 image, which is left aligned.</p>
 
-<p><img class="alignright wp-image-1" src="./../assets/images/150x150.png" /> As you can see the should be some space above, below, and to the right of the image. The text should not be creeping on the image. Creeping is just not right. Images need breathing room too. Let them speak like you words. Let them do their jobs without any hassle from the text. In about one more sentence here, we’ll see that the text moves from the right of the image down below the image in seamless transition. Again, letting the do it’s thang. Mission accomplished!</p>
+<p><img alt="" class="alignright wp-image-1" src="./../assets/images/150x150.png" /> As you can see the should be some space above, below, and to the right of the image. The text should not be creeping on the image. Creeping is just not right. Images need breathing room too. Let them speak like you words. Let them do their jobs without any hassle from the text. In about one more sentence here, we’ll see that the text moves from the right of the image down below the image in seamless transition. Again, letting the do it’s thang. Mission accomplished!</p>
 
-<p><img class="alignnone wp-image-1" src="./../assets/images/150x150.png" /></p>
```

Comment:
This PR also makes several changes like this one to get to 100% lighthouse report score for Accessibility wherever possible. The only remaining lighthouse warnings I observed were for the Prism syntax highlighting, which is third-party.